src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -7547,6 +7558,26 @@ private static boolean handleMelodramedary(String str, TagStatus status) {
     return true;
   }
 
+  private static boolean handlePatrioticEagle(String str, TagStatus status) {
```

Comment:
The net result being, this method is called and the combat properties are set - and that is tested.